examples/load_balancer/server.json

```diff
@@ -0,0 +1,17 @@
+{
+    "Info" : {
+        "LIST_SIZE": 2,
+        "policy": "random"
```

Comment:
we should make the capitalization of these fields consistent. I think changing to "list_size" would be better.